packages/gafl-webapp-service/src/pages/concessions/date-of-birth/__tests__/route.spec.js

```diff
@@ -1,31 +1,73 @@
 import { getData, validator } from '../route'
 import pageRoute from '../../../../routes/page-route.js'
 import { nextPage } from '../../../../routes/next-page.js'
+import { LICENCE_FOR } from '../../../../uri.js'
+import GetDataRedirect from '../../../../handlers/get-data-redirect.js'
 
 jest.mock('../../../../routes/next-page.js', () => ({
   nextPage: jest.fn()
 }))
 jest.mock('../../../../routes/page-route.js')
 
 describe('name > route', () => {
-  const mockTransactionCacheGet = jest.fn()
-
-  const mockRequest = {
+  const mockRequest = (statusGet = () => {}, transactionGet = () => {}) => ({
     cache: () => ({
       helpers: {
         transaction: {
-          getCurrentPermission: mockTransactionCacheGet
+          getCurrentPermission: transactionGet
+        },
+        status: {
+          getCurrentPermission: statusGet
         }
       }
     })
-  }
+  })
 
   describe('getData', () => {
     it('should return isLicenceForYou as true, if isLicenceForYou is true on the transaction cache', async () => {
-      mockTransactionCacheGet.mockImplementationOnce(() => ({ isLicenceForYou: true }))
-      const result = await getData(mockRequest)
+      const transaction = () => ({
+        isLicenceForYou: true
+      })
+      const status = () => ({
+        [LICENCE_FOR.page]: true
+      })
+      const result = await getData(mockRequest(status, transaction))
       expect(result.isLicenceForYou).toBeTruthy()
     })
+
+    it('should return isLicenceForYou as false, if isLicenceForYou is false on the transaction cache', async () => {
+      const transaction = () => ({
+        isLicenceForYou: false
+      })
+      const status = () => ({
+        [LICENCE_FOR.page]: true
+      })
+      const result = await getData(mockRequest(status, transaction))
+      expect(result.isLicenceForYou).toBeFalsy()
+    })
+  })
+
+  describe('redirectToStartOfJourney', () => {
+    it('should throw a redirect if not been to LICENCE_FOR page', async () => {
+      const transaction = () => ({
+        isLicenceForYou: true
+      })
+      const status = () => ({
+        [LICENCE_FOR.page]: false
+      })
+      const func = async () => await getData(mockRequest(status, transaction))
```

Comment:
Another comment - you don't need the async and await here. It'd work exactly the same if you did:
```
const func = () => getData(mockRequest(status, transaction))
```
The reason for this is that `func` would still return a Promise, which would then be `await`ed by the expect on the following line. So long as there's an `await` in there somewhere, it'll work fine